rmgpy/rmg/model.py

```diff
@@ -1931,6 +1932,6 @@ def areIdenticalSpeciesReferences(rxn1, rxn2):
     are identical, in either direction.
     """
 
-    return any([rxn1.reactants == rxn2.reactants and rxn1.products == rxn2.products, \
-            rxn1.reactants == rxn2.products and rxn1.products == rxn2.reactants
-            ])
+    return all([any([rxn1.reactants == rxn2.reactants and rxn1.products == rxn2.products,
```

Comment:
given there aren't many things being compared I think it would be clearer to just avoid one-lining it and do away with  the 'any' and 'all' in favor of 'and's and 'or's

Reply:
Done